lms/djangoapps/grades/tasks.py

```diff
@@ -35,12 +35,15 @@
 
 log = getLogger(__name__)
 
+COURSE_GRADE_TIMEOUT_SECONDS = 1200
```

Comment:
Were you able to find a newrelic trace that informed this choice of time limit? It seems kind of long.

Reply:
Discussed offline.